.github/workflows/semantic-release.yml

```diff
@@ -0,0 +1,27 @@
+name: Semantic release
+
+on:
+  repository_dispatch:
+    types: [semantic-release]
```

Comment:
This is triggered from our Blue bot. The PR for that work can be found here: https://github.com/fauna/blue/pull/1